automation/src/data/MockData.ts

```diff
@@ -0,0 +1,30 @@
+import faker from 'faker';
+
+/**
+ * Get fake user information.
+ */
+const user = {
+  firstname: faker.name.firstName(),
+  lastname: faker.name.lastName(),
+  job: faker.name.jobTitle(),
+  company: faker.company(),
+  words: faker.random.words(5),
```

Comment:
Let's keep the input the same every time, so `faker.lorem.words(5)`. I generally don't want any randomness on the input side in a test.